app/services/invite_schools.rb

```diff
@@ -303,6 +312,23 @@ def invite_not_opted_out_sits_with_all_validated_participants_for_nqt_plus_one
       end
   end
 
+  def invite_unpartnered_cip_sits_to_add_ects_and_mentors
+    School.unpartnered(Cohort.current.start_year)
+      .joins(:school_cohorts, :induction_coordinator_profiles)
+      .where(school_cohorts: { induction_programme_choice: "core_induction_programme" })
+      .where.missing(:ecf_participants)
+      .find_each do |school|
+        induction_coordinator = school.induction_coordinators.first
+
+        SchoolMailer.unpartnered_cip_sit_add_participants_email(
+          recipient: induction_coordinator.email,
+          induction_coordinator: induction_coordinator,
+          sign_in_url: sign_in_url_with_campaign(:add_participants_unpartnered_cip),
+          school_name: school.name,
+        ).deliver_later
+      end
+  end
```

Comment:
Does it send two emails to a coordinator that has two schools? Probably. Should we? Probably not :P

Reply:
Yes it will, I would vote that it should do since the email has the school name in the content, so it would remind them that they need to do it for both schools.

Reply:
Hm, can you double check with Shani / Tom that it's ok? 

I don't actually mind myself, your logic seems sound to me :)